app/src/main/res/values/strings.xml

```diff
@@ -138,19 +138,19 @@
     <string name="backup_call_log">Call history</string>
     <string name="backup_contacts">Local contacts</string>
     <string name="backup_section_user">Installed apps</string>
-    <!-- This text gets shown for apps that the OS did not try to backup for whatever reason e.g. no backup was run yet -->
+    <!-- This text gets shown for apps that the OS did not try to back up for whatever reason e.g. no backup was run yet -->
     <string name="backup_app_not_yet_backed_up">Waiting to back up…</string>
     <string name="restore_app_not_yet_backed_up">Was not yet backed up</string>
     <string name="backup_app_in_progress">Backing up</string>
     <string name="backup_app_success">Backed up</string>
     <string name="backup_app_warning">Backup warning</string>
     <string name="backup_app_was_stopped">Not backed up as it wasn\'t used recently</string>
```

Comment:
```suggestion
    <string name="backup_app_was_stopped">No backup needed</string>
```
"A backup for this already exists"?

Reply:
This is the string I hate the most, and one that confuses most people.

In this case, the system refuses to backup the application, because it's not running due to some reason, or because it may not have data to backup.

"No backup possible" or something along those lines might be better, but they're all confusing.

Reply:
I think that should be split into two cases,
one for errors that are internal to the Seedvault app,
one for errors to do with apps having nothing to back up, or actively/effectively preventing it.

"Not possible" isn't always the case with the former case.